src/Phpbb/TranslationValidator/Validator/FileListValidator.php

```diff
@@ -176,7 +176,7 @@ public function validate()
 				{
 					$this->output->addMessage(Output::NOTICE, 'Found additional file', $origin_file);
 				}
-				else if ($this->phpbbVersion == '3.1' && in_array($origin_file, array(
+				else if (($this->phpbbVersion == '3.1' || $this->phpbbVersion == '3.2') && in_array($origin_file, array(
```

Comment:
Just kill the version check here. That will allow .md files in 3.0 as well, but:
1. You don't validate that anymore
2. Since it's the goto way for later versions it doesn't really hurt.